src/model/ModelCore.i

```diff
@@ -27,13 +27,12 @@
 
   // ignore simulation objects for now, add back in with partial classes in ModelSimulation.i (TODO)
   %ignore openstudio::model::Model::foundationKivaSettings;
+  %ignore openstudio::model::Model::outputControlFiles;
   %ignore openstudio::model::Model::outputTableSummaryReports;
+  %ignore openstudio::model::Model::performancePrecisionTradeoffs;
   %ignore openstudio::model::Model::runPeriod;
-  %ignore openstudio::model::Model::weatherFile;
   %ignore openstudio::model::Model::yearDescription;
-  %ignore openstudio::model::Model::performancePrecisionTradeoffs;
-
-  %ignore openstudio::model::Model::outputControlFiles;
+  %ignore openstudio::model::Model::weatherFile;  
```

Comment:
If I take `SiteWaterMainsTemperature` for eg, it's swigged in ModelSimulation.i too.

so `%ignore openstudio::model::Model::SiteWaterMainsTemperature;` is needed for C# to swig correctly.